styles.css

```diff
@@ -0,0 +1,59 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+p {
+  margin: 20px 0;
+}
+
+h2 {
+  padding: 20px 0;
+}
+h1 {
+  padding: 20px 0;
```

Comment:
remember to leave spaces between rules, for better reading and clarity.

Reply:
- [ ] Please ensure to edit this comment as well to make it sound more polite. :ok_hand: 😊